--- activeheart-1.1.7/styleconf/activeheartconf.cpp.orig	2004-04-16 23:52:59.888188992 +0200
+++ activeheart-1.1.7/styleconf/activeheartconf.cpp	2004-04-17 00:24:34.462169968 +0200
@@ -1201,8 +1201,10 @@
 void ActiveHeartStyleConfig::slotLoadScheme()
 {
     QString schemeStr  = "/activeheart_" + schemeName->text() + ".kcm";
-    QString schemeFile = QDir::homeDirPath() + "/.qt" + schemeStr + "rc";
-    if (QFile::exists(schemeFile)) {
+    QStringList schemeFile( (QString) (QDir::homeDirPath() + "/.qt" + schemeStr + "rc"));
+    schemeFile << (QString) ("/usr/share/apps/kdisplay/color-schemes" + schemeStr + "rc");
+    for ( QStringList::Iterator it = schemeFile.begin(); it != schemeFile.end(); ++it ) {
+    if (QFile::exists(*it)) {
         int loadOK = QMessageBox::question(this,
 					   i18n("Load Scheme? - Active Heart Scheme Manager"),
 					   i18n("Do you load \"%1\"?").arg(schemeName->text()) + "\n"
@@ -1220,15 +1222,15 @@
 				     QMessageBox::NoButton);
 	    emit changed(true);
 	}
-    } else {
+/*  } else {
         QMessageBox::critical(this,
 			      i18n("Error - Active Heart Scheme Manager"),
 			      i18n("\"%1\" : No such scheme.").arg(schemeName->text()),
 			      QMessageBox::Cancel,
 			      QMessageBox::NoButton,
-			      QMessageBox::NoButton);          
+			      QMessageBox::NoButton);          */
+      }
     }
-
 }
 
 void ActiveHeartStyleConfig::slotSaveScheme()
@@ -1321,8 +1323,10 @@
 void ActiveHeartStyleConfig::searchScheme()
 {
     schemeListBox->clear();
-
-    QDir schemeDir(QDir::homeDirPath() + "/.qt", "activeheart_*.kcmrc");
+    QStringList schemeDirs (QDir::homeDirPath() + "/.qt");
+    schemeDirs << (QString) ("/usr/share/apps/kdisplay/color-schemes");
+    for ( QStringList::Iterator it = schemeDirs.begin(); it != schemeDirs.end(); ++it ) {
+    QDir schemeDir(*it, "activeheart_*.kcmrc");
     uint schemeCount = schemeDir.count();
     for (uint sidx = 0; sidx<schemeCount; sidx++) {
           QString schemeFile = schemeDir[sidx];
@@ -1333,7 +1337,7 @@
 		schemeListBox->insertItem(schemeFile);
 	  }
     }
-
+ }
 }
 
 #include "activeheartconf.moc"
